evap/results/tests/test_views.py

```diff
@@ -26,7 +26,7 @@ def setUpTestData(cls):
 
 
 class TestResultsSemesterCourseDetailView(ViewTest):
-    url = '/results/semester/2/course/21'
+    url = '/results/semester/1/course/1'
```

Comment:
because the course is created with the id 21, and the url accesses the course with the id 1, which may not exist at all.